histogram_file_manager/static/histogram_file_manager/js/components/file_actions.js

```diff
@@ -116,21 +122,21 @@ app.component('file-actions', {
                 )
                 .then((response) => {
                     console.log(response);
+                    this.success.push({
+                      id: this.message_id,
+                      message: `File id ${this.file_information.id} is being parsed.`
+                    });
+                    this.message_id += 1;
                 })
                 .catch((error) => {
                     console.error(error);
-                    this.errors.push(`${error}: ${error.response.data}`);
+                    this.errors.push({
+                      id: this.message_id,
+                      message: `${error}: ${error.response.data.detail}`
```

Comment:
I think think this should still be `error.response.data`. If using `error.response.data.detail`, when I try to re-parse [this](https://ml4dqm-playground.web.cern.ch/histogram_file_manager/?id=174538&data_era=&data_dimensionality=&granularity=&processing_complete=unknown&entries_total__gt=&entries_total__lt=&filepath__contains=) file (by also removing the `disabled` class on the Parse button), I get this:

![image](https://user-images.githubusercontent.com/8612771/211278761-526d5197-0e47-4fd7-9d78-6319745f3282.png)

Instead of:
![image](https://user-images.githubusercontent.com/8612771/211279122-8be153d9-04ae-4fab-80b6-5354b072803a.png)

Reply:
Thanks. I have fixed that in the latest commit.